modules/opensearch/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "environment" {
+  type        = string
+  description = "AWS tag to indicate environment name of each infrastructure object."
+}
+variable "dns_base_domain" {
+  type        = string
+  description = "DNS Zone name to be used in opensearch custom endpoint options."
+}
+variable "hosted_zone_id" {
+  type = string
+  description = "Route53 hosted zone id of the base domain."
+}
+variable "custom_endpoint_certificate_arn" {
+  type        = string
+  description = "The ACM cert arn to use with the custom endpoint."
+}
+variable "master_user_name" {
+  type = string
+  description = "Master username of opensearch user"
+}
+variable "master_user_password" {
+  type = string
+  description = "Master password of opensearch user"
```

Comment:
We should probably mark this as `sensitive = true`. https://learn.hashicorp.com/tutorials/terraform/sensitive-variables

Reply:
done